go-app-ussd_popi_user_data.js

```diff
@@ -375,39 +375,67 @@ go.app = function() {
                     }
                 },
                 next: function(content) {
-                    // TODO: Update ID on identitystore
-                    return 'state_updated';
+                    return 'state_create_identification_change';
                 }
             });
         });
 
-        self.add('state_change_passport', function(name) {
+        self.add('state_passport_origin', function(name) {
             return new ChoiceState(name, {
                 question: $('What is the country of origin of the passport?'),
                 choices: [
-                    new Choice('state_change_passport_zim', $('Zimbabwe')),
-                    new Choice('state_change_passport_moz', $('Mozambique')),
-                    new Choice('state_change_passport_mal', $('Malawi')),
-                    new Choice('state_change_passport_ng', $('Nigeria')),
-                    new Choice('state_change_passport_drc', $('DRC')),
-                    new Choice('state_change_passport_som', $('Somalia')),
-                    new Choice('state_change_passport_oth', $('Other'))
+                    new Choice('zw', $('Zimbabwe')),
+                    new Choice('mz', $('Mozambique')),
+                    new Choice('mw', $('Malawi')),
+                    new Choice('ng', $('Nigeria')),
+                    new Choice('cd', $('DRC')),
+                    new Choice('so', $('Somalia')),
+                    new Choice('other', $('Other'))
```

Comment:
@rudigiesler it looks like the nurseconnect and momconnect lines use different lists and the list in the hub app is only for the nurseconnect line. It looks to me like momconnect registrations for any countries other than 'mz' should have been marked as invalid :disappointed:
I'll extend the list of valid countries to include those in the momconnect list. We may want to investigate the work required to fix registrations that have been affected by this and fast-forward the mothers to the correct stage.